### DISCUSSION TITLE
how to run face api from mp4 file?

### DISCUSSION THREAD

**Discussion by psw7420fusion** [vladmandic/face-api#139] [Q&A] [answered]
i want make running face api from mp4 video file
i tried modify webcam.js but i don`t know how to run face api from video file

can i get some examples?

**Comment by vladmandic** [comment, marked as answer]
If all you want is to use video instead of webcam, simply remove section of the code from `webcam.js` that initializes webcam:
<https://github.com/vladmandic/face-api/blob/fb3836019f967169c4a198a4bcc2fd15f5e64f95/demo/webcam.js#L102-L130>

and replace with simple
```js
video.src = 'url_path_to_your_video';
```

**Reply by psw7420fusion** [reply]
Thanks for good libray and kind replies !
